frontend/src/components/Question.tsx

```diff
@@ -25,47 +37,36 @@ export const Question = () => {
         }}
       >
         <Typography variant="h5" sx={{ textAlign: "center" }}>
-          Problem Title
+          {title}
         </Typography>
         <Divider />
-        <Typography variant="body2" sx={{ textAlign: "justify" }}>
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent
-          efficitur et massa at molestie. In pharetra facilisis pulvinar. Ut
-          rutrum imperdiet lectus, a pulvinar ligula porta vel. Ut sollicitudin
-          diam id metus tincidunt convallis. Etiam scelerisque diam id
-          condimentum finibus. Sed viverra sem lorem, quis facilisis dui feugiat
-          nec. Vivamus in accumsan nisl. Nullam finibus finibus erat, ac
-          consequat nulla. Ut et metus at turpis sagittis efficitur. Nullam
-          condimentum mauris et imperdiet laoreet. Praesent dapibus, risus sed
-          sagittis tempus, elit ex rhoncus nibh, a congue elit orci ut urna.
-          Nullam nunc est, malesuada id magna ut, consectetur volutpat massa.
-          Donec dignissim orci nec fermentum ultricies.
-        </Typography>
-        <Typography variant="body1" sx={{ fontWeight: "bold" }}>
-          Example 1:
-        </Typography>
-        <Typography variant="body2" sx={{ textAlign: "justify" }}>
-          Aliquam sodales, odio quis sagittis dignissim, nibh ipsum tempus nibh,
-          ut ullamcorper nisl quam et elit. Phasellus sagittis libero nisl,
-          vitae convallis nibh tincidunt quis.
-        </Typography>
-        <Accordion
-          sx={{
-            boxShadow: "none",
-            "& .MuiAccordionSummary-root": { p: 0 },
-          }}
+        <Typography
+          component={"span"}
+          variant="body2"
+          sx={{ textAlign: "justify" }}
         >
-          <AccordionSummary expandIcon={<ExpandMore />}>
-            <Typography>Hint 1</Typography>
-          </AccordionSummary>
-          <AccordionDetails sx={{ p: 0 }}>
-            <Typography variant="body2">
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-              Suspendisse malesuada lacus ex, sit amet blandit leo lobortis
-              eget.
-            </Typography>
-          </AccordionDetails>
-        </Accordion>
+          {parse(DOMPurify.sanitize(content))}
+        </Typography>
+        <Stack direction="column">
+          {hints.map((hint, number) => (
+            <Accordion
```

Comment:
Also, expanding the accordions also seem to change the height of the entire room page. Can you see if you can check out which component is causing the expansion?

Reply:
Logged as bug in #209